be2-scala/src/main/scala/ch/epfl/pop/model/network/method/message/data/DataSchemaValidator.scala

```diff
@@ -0,0 +1,61 @@
+package ch.epfl.pop.model.network.method.message.data
+
+import ch.epfl.pop.model.network.method.message.data.ActionType.ActionType
+import ch.epfl.pop.model.network.method.message.data.ObjectType.ObjectType
+import ch.epfl.pop.pubsub.graph.Validator
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.networknt.schema.JsonSchema
+import org.slf4j.LoggerFactory
+
+import scala.collection.JavaConverters._
```

Comment:
~Careful JavaConverters are deprecated! You can use the CollectionConverters instead (see https://dotty.epfl.ch/api/scala/collection/JavaConverters$.html)~

Actually an even simpler way is to get rid of the import. You can achieve the same result (i beleive) using `wanted_collection(collection_to_convert)` instead of the asScala call (e.g. `List(list_to_convert)` if the collection is a list).

Reply:
Good to know! Thanks. That was part of some old PR but it's fixed now :+1: